code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -80,3 +84,33 @@
 		new /obj/item/stack/cable_coil(src,30,color)
 		new /obj/item/weapon/wirecutters(src)
 		new /obj/item/device/multitool(src)
+
+
+/obj/item/weapon/storage/toolbox/proc/update_force()
+	force = initial(force)
+	for (var/obj/item/I in contents)
+		force += I.w_class*1.3
+
+/obj/item/weapon/storage/toolbox/handle_item_insertion(obj/item/W as obj, prevent_warning = 0)
+	if (..(W, prevent_warning))
+		update_force()
+
+
+/obj/item/weapon/storage/toolbox/attack(mob/living/M as mob, mob/user as mob)
+	stunhit = 0
+	if (force > 14)
+		stunhit = 1

```

Comment:
Instastun and the usage of weaken for weapons is no bueno. The only thing that does it is the (currently experimental) telebaton. And I want to keep it that way. HALLOSS, and other mechanics, _which respect armour and other buffs and negatives,_ are to be preferred.

So this variable's use will need to be modified.